libkbfs/md_journal.go

```diff
@@ -510,6 +507,23 @@ func (j *mdJournal) convertToBranch(
 
 		prevID = newID
 
+		// If possible, replace the old RMD in the cache.  If it's not
```

Comment:
Ideally, the replacement of entries in the cache would only happen once we're sure that the branch conversion has finished, but I guess it doesn't matter too much.